plugins/wdc/wdc-nvme.c

```diff
@@ -1055,9 +1136,23 @@ static bool get_dev_mgment_cbs_data(int fd, __u8 log_id, void **cbs_data)
 	}
 	memset(data, 0, sizeof (__u8) * WDC_C2_LOG_BUF_LEN);
 
+	/* TODO:  get wdc uuid value */
```

Comment:
This presents a bit of a problem for including this feature at this time. Without a valid uuid to check, running this command is not going to get the correct result.

Reply:
Yeah, we set the default value to 1 since we thought the uuid index for WD would always be 1 and it would return the wdc values unless a uuid was found but now we understand that it could also be 0.  We are putting this case on hold until we get more firmware implementation details.  Closing this pull request.

Reply:
Isn't this something an OEM can change at their discretion?  The only thing an initial vendor can do is provide their own uuid value. Generally speaking, you don't want to run a uuid specific extension against the wrong device. It will be interpreted incorrectly, so I believe we need a valid uuid check for this to function reliably.